app/services/rapid_ready_for_decision/hypertension_special_issue_manager.rb

```diff
@@ -17,24 +19,17 @@ def add_special_issue
 
     private
 
-    def submission_data
-      @submission_data ||= JSON.parse(submission.form_json, symbolize_names: true)
-    end
```

Comment:
The `add_special_issue` method parses the submission form and then writes it back out to JSON, so there's not much value in `symbolize_names: true`. Also, make it consistent with how `Form526Submission` parses the form [here](https://github.com/department-of-veterans-affairs/vets-api/blob/3c48ee89440de8898b011079bffe547871a61096/app/models/form526_submission.rb#L154) so that we can maybe DRY it up later.